### PR TITLE
[POI] add speak related en-us

### LINKED ISSUE #98: [POI] Bot prompt includes more options in speech mode
#### What project is affected?
POI skill

#### What language is this in?
all

#### What happens?
user: “Find a coffee shop in Seattle”. 
bot says “I found the following” 

we expect the bot says “I’ve found {N} options, first one is {X} on {5th Avenue}, second is {Y} on {6th Avenue} and third is {Z} on {9th Avenue}. Which one would you like”?

### PR DESCRIPTION
**Pull request by xieofxie**
### Purpose
*What is the context of this pull request? Why is it being done?*

Close #98

### Changes
*Are there any changes that need to be called out as significant or particularly difficult to grasp? (Include illustrative screenshots for context if applicable.)*

* add speech related for en-us only
* add CardNames
* remove old speech
* fix from https://github.com/microsoft/botframework-solutions/pull/3006

### Tests
*Is this covered by existing tests or new ones? If no, why not?*

### Feature Plan
*Are there any remaining steps or dependencies before this issue can be fully resolved? If so, describe and link to any relevant pull requests or issues.*

#### General
- [ ] I have commented my code, particularly in hard-to-understand areas	
- [ ] I have added or updated the appropriate tests	
- [ ] I have updated related documentation

### CODE REVIEW

**Comment by xieofxie** [comment]
Hi @darrenj, what do you think of this? If it is good, I will apply these to other languages. Thanks!

File: skills/csharp/pointofinterestskill/Models/CardNames.cs [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace PointOfInterestSkill.Models
+{
+    public class CardNames
```
Comment (lzc850612) [inline skills/csharp/pointofinterestskill/Models/CardNames.cs]: make it static

File: skills/csharp/pointofinterestskill/Responses/FindPointOfInterest/FindPointOfInterestResponsesTexts.lg [hunk 2]
```diff
@@ -1,4 +1,15 @@
 # PointOfInterestDetails.Text(Data)
-- Here are details.
+- ${PointOfInterestDetailsTextAndSpeak1(Data)}
 
+# PointOfInterestDetailsTextAndSpeak1(Data)
+[Activity
```
Comment (lzc850612) [inline skills/csharp/pointofinterestskill/Responses/FindPointOfInterest/FindPointOfInterestResponsesTexts.lg]: This doesn't contain Text property. would it successfully generate an activity with Text property?

Comment (lzc850612) [reply skills/csharp/pointofinterestskill/Responses/FindPointOfInterest/FindPointOfInterestResponsesTexts.lg]: or do you just want Text property to be empty?

File: skills/csharp/tests/pointofinterestskill.tests/Flow/Strings/CardStrings.cs [hunk 3]
```diff
@@ -1,20 +1,18 @@
...
 // Licensed under the MIT License.
 
-using System;
-using System.Collections.Generic;
-using System.Text;
+using PointOfInterestSkill.Models;
 
 namespace PointOfInterestSkill.Tests.Flow.Strings
 {
     public class CardStrings
```
Comment (lzc850612) [inline skills/csharp/tests/pointofinterestskill.tests/Flow/Strings/CardStrings.cs]: let's make this class static

File: skills/csharp/pointofinterestskill/Models/CardNames.cs (hunk 1 above)
Comment (xieofxie) [reply skills/csharp/pointofinterestskill/Models/CardNames.cs]: made

File: skills/csharp/tests/pointofinterestskill.tests/Flow/Strings/CardStrings.cs (hunk 3 above)
Comment (xieofxie) [reply skills/csharp/tests/pointofinterestskill.tests/Flow/Strings/CardStrings.cs]: made

File: skills/csharp/pointofinterestskill/Responses/FindPointOfInterest/FindPointOfInterestResponsesTexts.lg (hunk 2 above)
Comment (xieofxie) [reply skills/csharp/pointofinterestskill/Responses/FindPointOfInterest/FindPointOfInterestResponsesTexts.lg]: Yes, then the text is null in generated activity.
It follows the previous visual design.

**Pull request merged**